node/Vagrantfile

```diff
@@ -0,0 +1,38 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    # Start with base vagrant box. This one is only 271 MB in size.
+    config.vm.box = "minimal/trusty64"
+
+    # Create forwarding ports for client-guest machine access via localhost.
+    config.vm.network :forwarded_port, guest: 80, host: 7331, auto_correct: true
+    config.vm.network :forwarded_port, guest: 6000, host: 9331, auto_correct: true
+
+    # Add the tty fix as mentioned in issue 1673 on vagrant repo
+    # To avoid 'stdin is not a tty' messages
+    # vagrant provisioning in shell runs bash -l
+    config.vm.provision "fix-no-tty", type: "shell" do |s|
+        s.privileged = false
+        s.inline = "sudo sed -i '/tty/!s/mesg n/tty -s \\&\\& mesg n/' /root/.profile"
+    end
+
+    # avoiding ssh key insert
+    config.ssh.insert_key = false
+
+    # ssh and gui mode
+    config.ssh.username = "vagrant"
+    config.ssh.password = "vagrant"
+
+    config.vm.provider "virtualbox" do |vb|
+      vb.gui = true
+    end
+
+    # naming the VM
+    config.vm.provider "virtualbox" do |v|
+      v.name = "nodeschool-vagrant"
+    end
```

Comment:
@koustuvsinha this is great but it has the caveat that you cannot spin up multiple simultaneous VMs with zeus. It will block the setup since there can only be one VM box with the same name. So either you wish to prioritise a nice name for your vm in virtualbox or being able to spawn multiple instances of the same vm.